src/nanoarrow/common/inline_array.h

```diff
@@ -914,6 +914,29 @@ static inline double ArrowArrayViewGetDoubleUnsafe(
   }
 }
 
+enum { ARROW_BINARY_VIEW_INLINE_SIZE = 12 };
```

Comment:
I originally had these as macros but the Cython generator did not like that. However, it doesn't appear to like the enum hack either.

Any ideas on what the best practice for the library here is @paleolimbot ?